models/bundle.js

```diff
@@ -15,15 +15,30 @@ function initModel(app) {
      *
      * @param {Version} version - the Version to update a bundle for.
      * @param {string} language - the language of the bundle to update, e.g. 'css' or 'js'.
-     * @param {string} brand [null] - the brand of the bundle to update, e.g. 'internal' (optional).
+     * @param {string} brand - the brand of the bundle to update, e.g. 'internal'.
      * @return {Bundle} - bundle information for the given version, language, and brand
      */
     async function updateBundleForVersion(version, language, brand = null) {
```

Comment:
As the JSDoc has been updated to state `brand` is no longer optional, should we remove the default value here?

```suggestion
    async function updateBundleForVersion(version, language, brand) {
```